projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-site-settings-endpoint.php

```diff
@@ -356,6 +356,9 @@ public function get_settings_response() {
 						)
 					);
 
+					require_lib( 'newsletter-categories' );
```

Comment:
`require_lib` isn't available on self-hosted sites, so this may cause issues?

Reply:
Thank you @jeherve, you were quick to this. 😁